DocumentVectorPipelineFunctions/BlobTriggerFunction.cs

```diff
@@ -72,21 +76,15 @@ private async Task HandleBlobCreateEventAsync(BlobClient blobClient)
             await this.ProcessCurrentBatchAsync(blobClient, cosmosDBClientWrapper, batchChunkTexts);
         }
 
-        this._logger.LogInformation("Created total chunks: '{documentCount}' of document.", totalChunksCount);
+        this._logger.LogInformation("Finished processing blob {0}, total chunks processed {1}.", blobClient.Name, totalChunksCount);
```

Comment:
When using `ILogger`, it's recommended to use meaningful names for replacements - you don't have to use `0`, `1`, etc the way you do in `string.Format()`.